### PR TITLE
updates for bounce rate

### PR DESCRIPTION
**Pull request by BillWagner**
- The "tour" and the "getting-started" were seeing bounce because beginning programmers saw both as day 0 content. Add links to the interactive tutorials.
- The keyword pages for a couple keywords that have multiple uses were bouncing. Update the opening to get the appropriate links in the introductory paragraph.
- Some other links drove traffic to these disambiguating pages, rather than a pasrticular use of the keyword.

### CODE REVIEW

File: docs/csharp/getting-started/index.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ helpviewer_keywords:
...
 author: rpetrusha
 ms.author: ronpet
-ms.date: 08/23/2017
+ms.date: 04/05/2019
+ms.custom: seoapril2019
 ---
 # Get started with C\#
 
-This section provides short, simple tutorials that let you quickly build an application using C# and .NET Core. There are getting started topics for Visual Studio 2017 and Visual Studio Code. You can build either a simple Hello World application or, if you have Visual Studio 2017, a simple class library that can be used by other applications.
+This section provides short, simple tutorials that let you quickly build an application using C# and .NET Core. There are getting started topics for Visual Studio 2017 and Visual Studio Code. These articles assume some programming experience. If C# is your first experience, try our [introduction to C#](../tutorials/intro-to-csharp/index.md) interactive tutorials.
```
Comment (rpetrusha) [inline docs/csharp/getting-started/index.md]: "If C# is your first experience" is awkward. Perhaps, "If you are new to programming, try..."

File: docs/csharp/language-reference/keywords/using.md [hunk 2]
```diff
@@ -11,13 +11,7 @@ ms.assetid: 124e1a63-2a4b-4132-b269-3b6d8d3ef72d
...
 # using (C# Reference)
 
-The `using` keyword has two major uses:
-
-- As a directive, when it is used to create an alias for a namespace or to import types defined in other namespaces. See [using directive](using-directive.md).
-
-- As a statement, when it defines a scope at the end of which an object will be disposed. See [using statement](using-statement.md).
-
-In addition, the [using static](using-static.md) directive lets you define a type whose static members you can access without specifying a type name.
+The `using` keyword has two major uses. The [using statement](using-statement.md) defines a scope at the end of which an object will be disposed. The using [directive](using-directive.md) creates an alias for a namespace or imports types defined in other namespaces. The related [using static directive](using-static.md) imports the members of a single class.
```
Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/using.md]: This is really 3 major uses.
I think that the original formatting is preferable, though -- at least, it makes the list-like content stand out more clearly.

File: docs/csharp/language-reference/keywords/default.md [hunk 3]
```diff
@@ -13,11 +13,7 @@ ms.assetid: 14c48aaa-7d35-4058-a1a4-f53353050579
 
 # default (C# Reference)
 
-The `default` keyword can be used in the `switch` statement or in a default value expression:
-
-- [The switch statement](switch.md): Specifies the default label.
-
-- [Default value expressions](../../programming-guide/statements-expressions-operators/default-value-expressions.md): Produces the default value of a type.
+The `default` keyword can be used in the [`switch` statement](switch.md) to specify the default label,  or in a [default value expression](../../programming-guide/statements-expressions-operators/default-value-expressions.md) to produce the default value of a type.
```
Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/default.md]:
```suggestion
The `default` keyword can be used in the [`switch` statement](switch.md) to specify the default label, or in a [default value expression](../../programming-guide/statements-expressions-operators/default-value-expressions.md) to produce the default value of a type.
```
Though it needs some modification, the original bulleted list is more readable.

**Review by rpetrusha — APPROVED** [review]
I've left a few comments, but I like these changes, @BillWagner. You can merge when you're ready.

File: docs/csharp/language-reference/keywords/default.md (hunk 3 above)
Comment (BillWagner) [reply docs/csharp/language-reference/keywords/default.md]: I think you're right.I switched back to lists.

**Pull request merged**